### DISCUSSION TITLE
geomap: dynamic symbols

### DISCUSSION THREAD

**Discussion by yosiasz** [grafana/grafana#57031] [Plugins]
dynamic geomap symbols

For example, if one manages different kind of devices and we want to plot this on a map it would be nice to be able to see clusters of specific device types on a map for a quick visual check/analysis based on the symbol without having to hover over each map point.

So if you have tel towers of a specific kind clustered in the NW, but you dont have enough of those in the SW you can get an idea with a quick glance of the symbols.

And these symbols could also be used as filter when clicking on them it dims the others and highlights only related ones.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
